R/helpers_kintr_Increments.R

```diff
@@ -0,0 +1,318 @@
+#' @importFrom knitr knit_print
+NULL
+
+# Increments ----
+
+#' Render a IncrementsRelative Object
```

Comment:
```suggestion
#' Render a `IncrementsRelative` Object
```
to avoid spelling problems as usual, same elsewhere

Reply:
Done.  Also for ``knit_print_CohortSize.R`, though it did not seem to be a problem.